### ISSUE TITLE
Lambda expression mapping didn't support source-directly-mapping

### ISSUE THREAD

**Issue by chhsiao90** [modelmapper/modelmapper#228]
PropertyMap supports mapping source directly to a destination property.
But ExpressionMap or expression mapping didn't support that.
I think it would be better to support this feature!

**Comment by JensRantil** [comment]
Is this part of #193?

**Comment by chhsiao90** [comment]
No, it's a follow up issue for lambda friendly api (#215) for next release.
